plugins/amqp/src/plugin.cpp

```diff
@@ -350,8 +350,16 @@ class amqp_engine {
     if (ret.library_error == AMQP_STATUS_TIMEOUT)
       return chunk_ptr{};
     // Now we're leaving the happy path.
-    if (ret.reply_type == AMQP_RESPONSE_LIBRARY_EXCEPTION
-        && ret.library_error == AMQP_STATUS_UNEXPECTED_STATE) {
+    TENZIR_DEBUG("leaving the AMQP happy path...");
```

Comment:
I don't think this log message adds any value over the one right below. Can you remove it?

Reply:
Sure. That was a leftover from @mavam's original